tutorial/README.md

```diff
@@ -0,0 +1,158 @@
+# GoBadge Tutorial
+
+## What you need
+
+    - GoBadge aka Adafruit PyBadge
+    - Personal computer with Go 1.18 and TinyGo installed, and a serial port.
```

Comment:
Latest go version is 1.19.1
Latest TinyGo version is 0.25.0. 

We should at least recommend 0.25.0 for the rp2040 flash issue that happened at gophercon.eu

Reply:
done